server/flyway/sql/V28__create_silva_application.sql

```diff
@@ -0,0 +1,58 @@
+-- Create silva_dev, silva_test and silva_prod applications
+INSERT INTO app_fam.fam_application (
+    application_name,
+    application_description,
+    app_environment,
+    create_user,
+    create_date
+)
+VALUES ('SILVA_DEV', ' (DEV)', 'DEV', CURRENT_USER, CURRENT_DATE),
```

Comment:
" (DEV)" Missing application_description for 'DEV' environment.
And the TEST/PROD description looks like SPAR (`Seed Planning and Registry Application`)

Reply:
Yep, copy paste error